src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,97 @@
+import { useMemo } from 'react';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { PersonLink } from '../PersonLink/PersonLink';
+import { Person } from '../../types';
+
+export type PropsPeopleTable = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<PropsPeopleTable> = ({ people }) => {
+  const { slug } = useParams();
+
+  const findParentOfPerson = useMemo(() => {
```

Comment:
small note: `useCallback` fits better for function memorization. Probably the mentor mistyped the hook name :)